### PR TITLE
[Merged by Bors] - feat: cleanup lint-style a bit

### PR DESCRIPTION
**Pull request by alexjbest**
Removing some linters and error codes that are not Lean 4 relevant

---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

**Review by Ruben-VandeVelde — APPROVED** [review]
Looks good to me

**Comment by Ruben-VandeVelde** [comment]
While you're here, you might consider removing the "!" exception in `long_lines_check` for porting comments that no longer exist

File: scripts/lint-style.py [hunk 1]
```diff
@@ -182,12 +154,7 @@ def import_only_check(lines, path):
         if imports[0] != "import":
             import_only_file = False
             break
```
[the remark this answers is not in the document]
Comment (alexjbest) [reply scripts/lint-style.py:156]: I agree, not sure what the "correct" way is but lets go with it

**Comment by alexjbest** [comment]
> While you're here, you might consider removing the "!" exception in `long_lines_check` for porting comments that no longer exist

nice catch thanks

**Comment by eric-wieser** [comment]
bors merge

I figured that fixing up the place where someone cheated with a `!` to avoid a long line is out of scope for this PR.

**Pull request closed without merging**